deepchecks/tabular/checks/model_evaluation/simple_model_comparison.py

```diff
@@ -390,8 +391,13 @@ def condition(result: Dict, include_classes=None, average=False, max_gain=None,
     task_type = result['type']
     scorers_perfect = result['scorers_perfect']
 
+    # If the depth of the nested scores dict is 2, average is not relevant and is set to True
+    inner_dict = scores[list(scores.keys())[0]]
+    inner_inner_dict = inner_dict[list(inner_dict.keys())[0]]
+    force_average = isinstance(inner_inner_dict, Number)
```

Comment:
why not just set average to this value? than you can remove this additional variable